src/NextGen/identity/identity.ts

```diff
@@ -1,48 +1,134 @@
 import { MPID, User } from '@mparticle/web-sdk';
 import { IMParticleComponent } from '../core/component';
+import { Context } from '@mparticle/event-models';
 
-// QUESTION: What is the diff between this and IdentityApiData?
-// TODO: sync this with android class
-export interface IdentityApiResult {}
-export interface IdentityApiRequest {}
+export interface IdentityApiResult {
+    user: User;
+    previousUser: User;
 
-// QUESTION: Should we mirror the way Android handles HTTP codes, or do it the way we've been doing it?
-// https://docs.mparticle.com/developers/sdk/android/idsync/#idsync-status-codes
-// https://docs.mparticle.com/developers/sdk/web/idsync/#error-handling
+    getUser(): User;
+    getPreviousUser(): User;
```

Comment:
This is for general discussion fro the group, but I don't see why there is a need to have a `user` and `previousUser` if we have the get methods for them.